pgoapi/pgoapi.py

```diff
@@ -22,28 +22,26 @@
 
 Author: tjado <https://github.com/tejado>
 """
-
-from __future__ import absolute_import
-
-import re
-import six
 import logging
 
-from . import __title__, __version__, __copyright__
-from pgoapi.rpc_api import RpcApi
-from pgoapi.auth_ptc import AuthPtc
-from pgoapi.auth_google import AuthGoogle
-from pgoapi.utilities import parse_api_endpoint
-from pgoapi.exceptions import AuthException, NotLoggedInException, ServerBusyOrOfflineException, NoPlayerPositionSetException, EmptySubrequestChainException, AuthTokenExpiredException, ServerApiEndpointRedirectException, UnexpectedResponseException
-
-from . import protos
 from POGOProtos.Networking.Requests_pb2 import RequestType
 
-logger = logging.getLogger(__name__)
+from . import __title__, __version__, __copyright__
+from .auth_google import AuthGoogle
```

Comment:
All of these relative imports don't make sense to be doing when the package layout is this simple